packages/core/src/model-types.ts

```diff
@@ -42,33 +41,24 @@ export type AwsBedrockChatModel = LiteralUnion<
   | 'anthropic--claude-3-sonnet'
   | 'anthropic--claude-3.5-sonnet'
   | 'anthropic--claude-3.7-sonnet'
+  | 'anthropic--claude-4-opus'
+  | 'anthropic--claude-4-sonnet'
   | 'amazon--titan-text-express'
   | 'amazon--titan-text-lite'
   | 'amazon--nova-pro'
   | 'amazon--nova-lite'
   | 'amazon--nova-micro'
+  | 'amazon--nova-premier'
 >;
 
-/**
- * AWS Bedrock models for embedding.
- */
-export type AwsBedrockEmbeddingModel = LiteralUnion<'amazon--titan-embed-text'>;
-
 /**
  * AI Core open source models for chat completion.
  */
 export type AiCoreOpenSourceChatModel = LiteralUnion<
-  | 'mistralai--mixtral-8x7b-instruct-v01'
   | 'mistralai--mistral-large-instruct'
   | 'mistralai--mistral-small-instruct'
   | 'meta--llama3.1-70b-instruct'
```

Comment:
[pp] `meta--llama3.1-70b-instruct` is also marked as deprecated but will only be retired by 30th September, so we can wait a bit longer here.

Reply:
In the SAP Notes, it says `mistralai--mistral-small-instruct` is the replacement in the "Recently Retired Models" table.

Reply:
Yes, for `meta--llama3.1-70b-instruct` if there is no replacement, we remove but also not suggest anything.